[IMP] WhatsApp: Permanent Tokens + Code Errors (#12932)
odoo/documentation: content/applications/productivity/whatsapp.rst

The WhatsApp connector supports two flows: company initiated, and customer initiated. A company can
initiate a discussion by sending a template to one or more people. Once the template is sent, the
recipient can answer in order to trigger a discussion between the sender and the receiver (a

> *Discuss* chat window will appear if the customer answers within 15 days).

Reviewer: Bold app name and reword to avoid future-tense
```suggestion
**Discuss** chat window appears if the customer answers within 15 days).
```

for Developers` dashboard.

.. note::

>    If the Meta business account is prohibited from advertising, claiming an app will not be allowed. To
>    resolve this issue navigate to `<https://business.facebook.com/business>`_ for assistance.

Reviewer: Break line earlier to avoid going over 100 chars and reword to avoid future-tense
```suggestion
   If the Meta business account is prohibited from advertising, claiming an app is not allowed. To
   resolve this issue navigate to `<https://business.facebook.com/business>`_ for assistance.
```

fields` section. Click :guilabel:`Manage` and when the pop-up window appears, check the boxes in the

> :guilabel:`Subscribe` column for the following *field names*:

Reviewer: Nitpick– unecessary formatting
```suggestion
:guilabel:`Subscribe` column for the following field names:
```

Assets now must be added to the system user and then a permanent token can be generated.

.. warning::

>    This is a mandatory step. If the permanent token is not added, the Odoo database will show a

Reviewer: Avoid future tense :)
```suggestion
   This is a mandatory step. If the permanent token is not added, the Odoo database shows a
```

>    :ref:`token error <whatsapp/token_error>`.

Author: A requested edit from Jose Bravo (job)

   produce an error The :guilabel:`Phone Field` should always be set to the `Phone` or `Mobile`
   model.

> To search available fields, type in the front-end name in the :guilabel:`Search...` box. This will
> find a result from all of the available fields for the model (:guilabel:`Applies to`) that the
> template is created for.

Reviewer: We no longer need formatting for the search, and future tense
```suggestion
To search available fields, type in the front-end name in the search box. This finds a result from
all of the available fields for the model (:guilabel:`Applies to`) that the template is created for.
```
